src/components/pill/pill-classic.style.js

```diff
@@ -87,6 +87,30 @@ export default (colorVariant, inFill, isDeletable) => {
 
     ${!isDeletable && css`
       padding: 2px 7px;
+
+      ${size === 'small' && css`
+        font-size: 10px;
+
+        button {
+          .carbon-icon {
+            &:before {
+              font-size: 8px;
+            }
+          }
+        }
+      `}
+
+      ${size === 'medium' && css`
+        font-size: 12px;
+        min-width: 38px;
+        padding: 4px 8px 4px 8px;
```

Comment:
We can rewrite it to: `padding: 4px 8px`

Reply:
updated.